File: ruby/spec/parser/copy.spec.rb
input = <<~DOC.strip
template:
key<template
key<<template
key < template
key << template
key     < template
key     << template
     key      < template
     key      << template
DOC

describe Enolib::Parser do
  describe 'Copy operations' do
    context = Enolib::Context.new(input)

    it 'parses as expected' do
      expect(context.document).to match_snapshot
    end
  end
end

File: javascript/specs/tokenizer/copy.spec.js
const { inspectTokenization } = require('./util.js');

const input = `
template:

key < template
key    < template
     key     < template
`.trim();

describe('Copy tokenization', () => {
  it('performs as specified', () => {
    expect(inspectTokenization(input)).toMatchSnapshot();
  });
});
